gis/GPXLab/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           qmake5 1.0
+
+github.setup        BourgeoisLab GPXLab v0.7.0
```

Comment:
We don't want the v in the version number, use this incantation instead:

`github.setup        BourgeoisLab GPXLab 0.7.0 v
`

Reply:
Fixed.